src/store/rule/service.ts

```diff
@@ -23,8 +23,10 @@ export default {
     );
   },
   async editRule(payload:RequestPayload<Rule>): Promise<ApiResponse<Rule>> {
+    const id = payload.payload.ID
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
    const id = payload.payload.ID;
```